.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,59 @@
+### Description, Motivation & Context
+
+<!-- Help readers to understand why this is relevant -->
+
+### Related issues
+
+<!-- Please link any related issues here. 
+     Use "Closes #123" to reference issues that should be closed automatically when this is merged. -->
+
+### Links and data
+
+**Relevant OSM Wiki links:**
+- …
+
+**Relevant tag usage stats:**
+> …
+<!-- E.g., Numbers from [Taginfo](https://taginfo.openstreetmap.org/) or [local Taginfo](https://taginfo.geofabrik.de/) -->
+<!-- E.g., A link to https://taghistory.raifer.tech -->
+
+### Checklist and Test-Documentation template.
+
+Once the preview is generated, **copy the following template and create a comment** to add the info.
```

Comment:
Alternatively, we could encourage people to come back later and edit the PR description, instead of copy-pasting this template into a separate comment. 

The reason I suggest this is because, it may lead to PRs that contain the empty template in the PR description, and the completed template in a separate comment. 

But this is just a hypothetical problem, if it actually causes confusion we could revisit this in the future

Reply:
Agreed. I wrote it like this because we will get notifications on new comments … but not on changes to the existing description. But we might need to change that if it does not work…

Reply:
FYI: I reworked this so the initial message is less cluttered.